main.go

```diff
@@ -85,6 +85,10 @@ func main() {
 	if config.Execute != "" {
 		input, err = executeCommand(config)
```

Comment:
Is it an input or an output?

Reply:
In the main branch is how it was:

https://github.com/charmbracelet/freeze/blob/335467362a96eec3e0ff6514a27006dfccaa7b62/main.go#L85-L93

Do I change it to be `output` variable?

Reply:
I think so, look at execute method, the variable is named output, no?

Reply:
Yeah, it's called `out`, so we can call it that. After all, it exits once `printErrorFatal` call ends.

Reply:
Just to update on why I changed this back to `input` - it's typically an input file that's being provided to the program whose value is checked further down, so it must be set. If this is renamed, the tests fail and we get an error for all `--execute` commands that don't get an error back. For example:

![image](https://github.com/user-attachments/assets/c5592bc7-5911-4eb8-baec-0d7116b66888)